homework1/ParallelMatrixMultiplication/ParallelMatrixMultiplication/FileMatrixMultiplicator.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.IO;
+
+namespace ParallelMatrixMultiplication
+{
+    /// <summary>
+    /// represents a class for multiplication of two matrices written in files
+    /// </summary>
+    public class FileMatrixMultiplicator
+    {
+        private MultiplicationStrategy strategy;
+
+        /// <summary>
+        /// a strategy of multiplication
+        /// </summary>
+        public MultiplicationStrategy Strategy
+        {
+            get => strategy;
+            set => strategy = value?? throw new ArgumentNullException();
```

Comment:
```suggestion
            set => strategy = value ?? throw new ArgumentNullException();
```
Бинарные операторы все с пробелами пишутся